src/components/TransactionFlow/TxReceiptTotals.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import feeIcon from '@assets/images/icn-fee.svg';
+import sendIcon from '@assets/images/icn-send.svg';
+import sentIcon from '@assets/images/icn-sent.svg';
+import { Amount, AssetIcon } from '@components';
+import { getFiat } from '@config/fiats';
+import translate from '@translations';
+import { ExtendedAsset, ISettings } from '@types';
+import { convertToFiat, fromWei, totalTxFeeToWei, Wei } from '@utils';
+
+interface Props {
+  asset: ExtendedAsset;
+  assetAmount: string;
+  baseAsset: ExtendedAsset;
+  settings: ISettings;
+  gasUsed: string;
+  gasPrice: string;
+  value: string;
+  assetRate?: number;
+  baseAssetRate?: number;
+}
+
+export const TxReceiptTotals = ({
```

Comment:
Is it safe to use floats on this component ?

Reply:
Which part are you worried about?

Reply:
The fee part, from `feeFormated` to `totalEtherFormatted`, idk if for fiats amounts it's a problem since there isn't as much decimals